AutoRest/Generators/CSharp/CSharp.Tests/Expected/Mirror.Primitives/Models/Product.cs

```diff
@@ -168,6 +176,11 @@ public Product(int? integer = default(int?), int? intProperty = default(int?), l
 
         /// <summary>
         /// </summary>
+        [JsonProperty(PropertyName = "base64urlArray")]
```

Comment:
Sorry for not responding earlier. That's very odd since there is no other way for json.net to know to use this converter unless it's globally declared (which is not something you want to do in this case)